src/styles/pages/press-releases.scss

```diff
@@ -0,0 +1,138 @@
+@import "../common";
+
+.PingCAP-PressReleases {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.PressReleases {
+  $padding-top: 60px;
+  $padding-bottom: 120px;
+  $date-margin-bottom: 2rem;
+  $img-margin-right: 75px;
+
+  padding: {
+    top: $padding-top;
+    bottom: $padding-bottom;
+  };
+
+  @include mobile {
+    background-color: #F7F8F9;
```

Comment:
Use color variable to set the backgroud-color.

You need to check the color you want to assign to an attribute whether has been defined on `_variable.scss` or not, if it is defined, use the variable name.